commands/manifest/template.go

```diff
@@ -118,3 +109,45 @@ func templateCmd(c *cli.Context) error {
 
 	return nil
 }
+
+func getTemplatesManifests(m dx.Manifest) (string, error) {
+
+	templatemanifests, err := templateChart(m)
```

Comment:
`templatemanifests` lost the `s` but given the history of `templatesManifests` and the proposed `getManifests` we could just call this var `manifests`